File: miraculous-backend/src/main/java/com/example/miraculousbackend/dto/CommentRetrievalDto.java
package com.example.miraculousbackend.dto;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CommentRetrievalDto {

    private Long id;
    private Long authorId;
    private String authorLogin;
    private String dateOfComment;
    private String role;
    private String text;
    private Long numberOfLikes;
    private String commentAuthorPictureUrlName;
}

File: miraculous-frontend/miraculous-ui/src/dto/comment-retrieval-dto/comment-retrieval-dto.ts
export class CommentRetrievalDto {

  id: number;
  authorId: number;
  authorLogin:string;
  dateOfComment: string;
  role: string;
  text: string;
  numberOfLikes: number;
  commentAuthorPictureUrlName: string;


  constructor(id?: number, authorId?: number, authorLogin?: string, dateOfComment?: string, role?: string, text?: string,
              numberOfLikes?: number, commentAuthorPictureUrlName?: string) {

    this.id = id;
    this.authorId = authorId;
    this.authorLogin = authorLogin;
    this.dateOfComment = dateOfComment;
    this.role = role;
    this.text = text;
    this.numberOfLikes = numberOfLikes;
    this.commentAuthorPictureUrlName = commentAuthorPictureUrlName;
  }
}
